pyproject.toml

```diff
@@ -23,7 +23,8 @@ dependencies = [
     'google-cloud-storage',
     'pyminizip',
     'aind-codeocean-api==0.1.0',
-    'aind-data-schema==0.12.9'
+    'aind-data-schema==0.12.9',
+    'aind-metadata-service[client]'
```

Comment:
The current version of aind-metadata-service[client] pins the version of aind-data-schema to 0.12.9 right now. I think this pr will be successful, but there might be potential issues in the future. The aind-metadata-service[client] doesn't need a dependency on aind-data-schema, so I'll fix it there, but the version of aind-metadata-service might need to be upgraded in the future.